### ISSUE TITLE
(BUG) "NMS: Next" - Ship changes don't appear to work

### ISSUE THREAD

**Issue by Vaygrim** [goatfungus/NMSSaveEditor#5]
Playing with the new NEXT update, along with version 1.5.1 of your editor. I've been able to adjust my character's money, nanites, etc. One big appear that appears to be broken however, is making changes to my ship. Unlocked slots to take effect when saved. Ship design / type don't change, etc.

Am I just doing something wrong, or are these features not implemented yet?

**Comment by goatfungus** [comment]
If you are trying to change the ship seed, please bear in mind that old seeds generated before NEXT will not look the same as the models have changed. Could this be your problem?

**Comment by Vaygrim** [comment]
So I got my newly created character, which is in slot 1, hit "Save As" and saved it to Slot 2. Start up game, bingo I've got two characters now and I can use Slot 2 for save file editing experiments.

Turns out all the changes I was doing to the Slot 2 copy... was actually taking affect in the Slot 1 file? LOL

I don't know if that's a ME screw up, or what.. but it's definitely not what I listed on this issue's subject line. SO.. I'm going to close this. Thanks, sorry to bother!
